utils/prepare_test_env.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -o errexit
+
+# Setup virtualenv
+virtualenv venv
+source venv/bin/activate
+
+# Ensure you have the latest version of pip
+pip install --upgrade pip
+
+# Install packs requirements
+pip install -r requirements.txt
+pip install -r requirements-test.txt
+
+# Checkout and install st2 requirements
+git clone https://github.com/StackStorm/st2.git --depth 1 --single-branch --branch v1.5.1 ./st2
```

Comment:
Ok. So we need to have both. 

But we still need to have these two versions synced with what we're running at SNPSEQ -- or what we want to run. So at least it must be easy to adjust.